src/libostree/ostree-fetcher-soup.c

```diff
@@ -1092,10 +1086,11 @@ on_request_sent (GObject        *object,
                   code = G_IO_ERROR_BROKEN_PIPE;
 #else
                   code = G_IO_ERROR_CONNECTION_CLOSED;
-#endif
```

Comment:
Did you mean to move this `#endif`?

Reply:
argh, no, the problems of squashing two diffs